src/components/Navrail/Navrail.tsx

```diff
@@ -105,27 +104,27 @@ export const Navrail: FC = () => {
   return (
     <div className="dark">
       <Sidebar collapsible="icon">
-        <SidebarHeader className={`${styles.Header}`}>
-          {state === 'expanded' && (
-            <SidebarMenuButton className={`${styles.Gem}`} asChild>
-              <div onClick={() => navigate(ROUTES.OVERVIEW)}>
-                <Gem />
-              </div>
-            </SidebarMenuButton>
-          )}
-
+        <SidebarHeader className="flex-row-reverse justify-between">
```

Comment:
this styling makes it so that the sidebar collapse button doesn't snap to the left when the button is clicked:

https://github.com/user-attachments/assets/514f021a-d572-4314-a93e-608bc35b765b